<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInventoryMenu</class>
 <widget class="QDialog" name="EditInventoryMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="leftpane" native="true">
         <layout class="QVBoxLayout" name="invMenuItemslayout">
          <item>
           <widget class="QLabel" name="invmenuItemsLabel">
            <property name="text">
             <string>Items Used</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="invMenuItemsSearch">
            <property name="placeholderText">
             <string>Search Used Inventory Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="invMenuItemsControlsLayout" native="true">
            <layout class="QHBoxLayout" name="usedItemControls">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="itemAmountOverrideLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Item Amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="itemAmountOverride">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeItem">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Remove Item</string>
               </property>
               <property name="icon">
                <iconset theme="delete">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="invMenuItems">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="invMenuItemsRemove">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Remove Selected Item</string>
            </property>
            <property name="icon">
             <iconset theme="draw-arrow-forward">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="rightpane" native="true">
         <layout class="QVBoxLayout" name="invItemsLayout">
          <item>
           <widget class="QLabel" name="invItemsLabel">
            <property name="text">
             <string>All Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="invItemsSearch">
            <property name="placeholderText">
             <string>Search Inventory Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="invItems">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="invItemsAdd">
            <property name="text">
             <string>Add Selected Item</string>
            </property>
            <property name="icon">
             <iconset theme="draw-arrow-back">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditInventoryMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
